compiler/luci-interpreter/src/kernels/CMakeLists.txt

```
@@ -1,234 +1,60 @@
-find_package(Threads REQUIRED)
+if (NOT LUCI_MICRO)
+  find_package(Threads REQUIRED)
+  set(KERNEL_REGISTER_FILE "${LUCI_INTERPRETER_SOURCE_DIR}/KernelsToBuild.lst")
+else ()
+  set(KERNEL_REGISTER_FILE "${LUCI_INTERPRETER_SOURCE_DIR}/micro/KernelsToBuild.lst")
+endif ()
```

Maybe add `KernelsToBuild.lst` into appropriate pal directory?

Then this code could look like:
```suggestion
set(KERNEL_REGISTER_FILE ${LUCI_INTERPRETER_PAL_DIR}/KernelsToBuild.lst)
```